<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_zouhour</class>
 <widget class="QMainWindow" name="gestion_zouhour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/res/img/hand-painted-watercolor-background-with-sky-clouds-shape_24972-1095.jpg);
background-color: rgb(255, 170, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="ajouterE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>741</width>
      <height>551</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Employ√©s</string>
     </attribute>
     <widget class="QLineEdit" name="rech_emp">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="table_emp">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>661</width>
        <height>311</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#d54e21;&quot;&gt;Gestion Employes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stat_emp">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>410</y>
        <width>121</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/res/img/stat1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="emp_sel_col">
      <property name="geometry">
       <rect>
        <x>591</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prenom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mail</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Naissance</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Occupation</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Missions</string>
     </attribute>
     <widget class="QTableView" name="table_mission">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>651</width>
        <height>361</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#d54e21;&quot;&gt;Gestion missions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech_mission">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="mission_sel_col">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Description</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>ajouterE</string>
     </attribute>
     <widget class="QWidget" name="ajouter_e" native="true">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>391</width>
        <height>471</height>
       </rect>
      </property>
      <widget class="QDateEdit" name="naissance">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>280</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>240</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
      <widget class="QComboBox" name="mission">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>360</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>360</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Mission</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>280</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Date Naissance</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>200</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>430</y>
         <width>156</width>
         <height>23</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>160</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>200</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QComboBox" name="occupation">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>320</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Journaliste</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Agent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Admin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string> Ecrivain</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="mail">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>240</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>160</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>320</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Occupation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#0000ff;&quot;&gt;Details Employe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>ajouterM</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>381</width>
        <height>441</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#0000ff;&quot;&gt;Details Mission&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="description">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>210</y>
         <width>271</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>180</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="buttonBox_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>360</y>
         <width>156</width>
         <height>23</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="type">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>140</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>100</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd_emp"/>
   <addaction name="actiondelete_emp"/>
   <addaction name="separator"/>
   <addaction name="actionadd_mission"/>
   <addaction name="actiondelete_mission"/>
  </widget>
  <action name="actionadd_emp">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/res/img/supp.jfif</normaloff>:/res/img/supp.jfif</iconset>
   </property>
   <property name="text">
    <string>veuillez ajouter un employe</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/res/img/supp.jfif&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actiondelete_emp">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/res/img/ajout.png</normaloff>:/res/img/ajout.png</iconset>
   </property>
   <property name="text">
    <string>veuillez supprimer un employe</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/res/img/ajout.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionadd_mission">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/res/img/images (2).jfif</normaloff>:/res/img/images (2).jfif</iconset>
   </property>
   <property name="text">
    <string>add_mission</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/res/img/images (2).jfif&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actiondelete_mission">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/res/img/images (1).jfif</normaloff>:/res/img/images (1).jfif</iconset>
   </property>
   <property name="text">
    <string>delete_mission</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/res/img/images (1).jfif&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionloupe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/res/img/loupe1.jfif</normaloff>:/res/img/loupe1.jfif</iconset>
   </property>
   <property name="text">
    <string>loupe</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/res/img/loupe1.jfif&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
